changelog/v0.2.0/k8s-1.24+.yaml

```
@@ -0,0 +1,21 @@
+changelog:
+  - type: BREAKING_CHANGE
+    issueLink: https://github.com/solo-io/gloo/issues/7071
+    resolvesIssue: false
+    description: Clustername override
```

```suggestion
    description: Use annotations to store/retrieve cluster names, to support kubernetes versions 1.24 and later where the `metadata.clusterName` field does not exist.
```
not sure if this changelog show up anywhere but let's put a more detailed comment